### PR TITLE
check logging namespace for actual resources

### PR DESCRIPTION
**Pull request by jcantrill**
Check for the logging namespace with resources in case there is just an empty namespace

### CODE REVIEW

File: hack/testing/entrypoint.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
 source "${OS_O_A_L_DIR}/hack/testing/util.sh"
 
 LOGGING_NS=openshift-logging
-if oc get project logging -o name > /dev/null ; then
+if oc get project logging -o name > /dev/null && [ $(oc get dc -n logging -o name | wc -l) -gt 0 ]  ; then
```
Comment (ewolinetz) [inline hack/testing/entrypoint.sh]: isn't the `oc get` command missing an object type?

**Comment by richm** [comment]
@jcantrill I say we force merge this to unblock CI - and force merge on es5.x branch as well

**Pull request merged**